contracts/modules/delegates/IexecPoco2Delegate.sol

```diff
@@ -136,7 +139,8 @@ contract IexecPoco2Delegate is IexecPoco2, DelegateBase, IexecERC20Core, Signatu
         contribution.enclaveChallenge = _enclaveChallenge;
         task.contributors.push(_msgSender());
 
-        lockContribution(task.dealid, _msgSender());
+        // Lock contribution.
+        lock(_msgSender(), deal.workerStake);
```

Comment:
Remove `lockContribution` and methods like that (or add TODO but coverage will eventually complains)

Reply:
Done
[Remove unused ERC20ore functions](https://github.com/iExecBlockchainComputing/PoCo/pull/63/commits/bcb9ddab2978cfd774c045f54c7c7debf8708abb)
[Clean IexecERC20Core](https://github.com/iExecBlockchainComputing/PoCo/pull/63/commits/f9236b228c2c4bdd106eaa9913d9922e10b043f5)